lib/ds.ts

```diff
@@ -43,7 +43,12 @@ export interface Config {
 const validateGetRelationParams = (
   params: PartialMessage<RelationIdentifier>
 ) => {
-  validateObjectRef(params.object!, "object");
+  try {
+    validateObjectRef(params.object!, "object");
+  } catch (error) {
+    validateObjectRef(params.subject!, "subject");
+    validateObjectType(params.object!, "object");
+  }
   validateObjectType(params.subject!, "subject");
```

Comment:
Shouldn't we have the same validation logic here for `params.subject`?

Reply:
And to be honest, I'm not sure it makes sense to have these kinds of validations in the SDK. I think this should happen on the backend.

Reply:
@ronenh I totally agree, the issue right now is that the backend validations are not providing the best feedback. Thi is the third time i update this validation, based on the consumer needs so definitely this is not the right place to do it,